doc/input-output-reference/src/overview/group-internal-gains-people-lights-other.tex

```diff
@@ -827,20 +802,20 @@ \subsection{Simplified ASHRAE 55 Warnings}\label{simplified-ashrae-55-warnings}
 \item
   Eliminate occupancy when conditioning equipment is off.
 \item
-  Note that the ASHRAE graph lower limit is (19.6C to 21.7C) -- heating setpoints may need to be nearer 22.2C (72F) than 21.1C (70F).
+  Note that the ASHRAE graph lower limit is (19.6$^\circ$C to 21.7$^\circ$C)---heating setpoints may need to be nearer 22.2$^\circ$C (72$^\circ$F) than 21.1$^\circ$C (70$^\circ$F).
 \item
```

Comment:
I thought we were using \deg or something like that but this does give a good degree symbol and Google shows this is one method.

![image](https://user-images.githubusercontent.com/5778156/189942645-2f11c4b7-d4e4-4891-94eb-21933563ef70.png)